yomigeek/mtracker-app
# #158281077 Implement Admin Approve Request

#### What does this PR do?
Implement Admin Approve Request

#### Description of Task to be completed?
Admin should be able to approve a user request

#### How should this be manually tested?
Go to the url https://mtrack-app.herokuapp.com/login.html
Enter the required credentials
If successful the user dashboard page is displayed 
On the dashboard an admin clicks the View button
On the View request page an admin clicks on the approve button

#### Any background context you want to provide?
None

####  What are the relevant pivotal tracker stories?
story type: feature
story id: 158281077

#### What are the relevant Github Issues?
None

#### Questions:
None
